Basic-Car-Maintenance/Shared/Utilities/Constants.swift

```diff
@@ -16,3 +16,34 @@ enum FirestoreCollection {
 enum FirestoreField {
     static let userID = "userID"
 }
+
+enum SFSymbol {
+    //Tabview icons
+    static let dashboard = "list.dash.header.rectangle"
+    static let odometer = "gauge"
+    static let settings = "gear"
+
+    //Navigation header icons
```

Comment:
```suggestion
    // Navigation Items
```

Fix the spacing for the rest, and remove the word "icons"